cmd/node.go

```diff
@@ -118,13 +122,36 @@ func (node *Node) Start() {
 		lib.StartDBSummarySnapshots(node.chainDB)
 	}
 
+	// Setup postgres using a remote URI
+	var db *pg.DB
+	if node.Config.PostgresURI != "" {
+		options, err := pg.ParseURL(node.Config.PostgresURI)
+		if err != nil {
+			panic(err)
+		}
+
+		db = pg.Connect(options)
+	}
+
+	if db != nil {
+		node.Postgres = lib.NewPostgres(db)
+
+		// Make sure we're migrated
+		migrate.LoadMigrations()
```

Comment:
Add comments explaining, in detail, how the migrations work here. For example, what is LoadMigrations? Why do we need to call that before calling Run()? Where are the migrations stored and how are they being loaded? Make sure you give the devs all the info they need to trace everything here.